bindings/cs/Protean-cs/Protean.Net/VariantBase.cs

```diff
@@ -529,6 +550,11 @@ public static string ToString(TimeSpan arg)
             return $"{(arg < TimeSpan.Zero ? "-" : "")}{arg:hh\\:mm\\:ss}{(arg.Milliseconds != 0 ? arg.ToString(@"\.FFF") : "")}";
         }
 
+        public override int GetHashCode()
+        {
+            return Value.GetHashCode();
```

Comment:
Shouldn't we be including the `Type` in the hash?

Reply:
Not sure why GitHub doesn't pick up that his has been updated